### PR TITLE
Added read & write for docket, bill, ship, kelvin

### PR DESCRIPTION
**Pull request by willbach**

### CODE REVIEW

**Review by dr-frmr — CHANGES_REQUESTED** [review]
Logic looks good, just a style comment

File: app/ziggurat.hoon [hunk 1]
```diff
@@ -439,7 +439,12 @@
         %save-file
       :_  state
       :~  =-  [%pass /save-wire %arvo %c -]
-          [%info `@tas`project.act %& [file.act %ins %hoon !>(text.act)]~]
+          ?+  (rear file.act)  [%info `@tas`project.act %& [file.act %ins %hoon !>(text.act)]~]  :: assume hoon as the default file type
+            %ship  [%info `@tas`project.act %& [file.act %ins %ship !>(;;(@p q:(slap !>(~) (ream text.act))))]~]
+            %bill  [%info `@tas`project.act %& [file.act %ins %bill !>(;;((list @tas) q:(slap !>(~) (ream text.act))))]~]
+            %kelvin  [%info `@tas`project.act %& [file.act %ins %kelvin !>(;;([@tas @ud] q:(slap !>(~) (ream text.act))))]~]
+            %docket-0  [%info `@tas`project.act %& [file.act %ins %docket-0 !>((need (from-clauses:mime:dock ;;((list clause:dock) q:(slap !>(~) (ream text.act))))))]~]
+          ==
```
[the remark this answers is not in the document]
Comment (dr-frmr) [reply app/ziggurat.hoon:447]: Actually should probably factor out the `q:(slap !>(~) (ream text.act))` as well, by assigning that value to a face above the switch with `=/`

[the remark this answers is not in the document]
Comment (willbach) [reply app/ziggurat.hoon]: I just made those changes and confirmed that it is compiling

**Review by dr-frmr — APPROVED** [review]

**Comment by hosted-fornet** [comment]
Thanks for the bugfix!

**Pull request merged**